### PR TITLE
{Release} Upgrade to Azure CLI 2.26.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,136 @@
...
+* `az webapp/functionapp config access-restriction remove`: Remove service endpoints are case-insensitive (#18024)
+* `az webapp config access-restrictions add`: Skip validation if user does not have access to get service tag list. (#18527)
+* Add support for Linux Consumption and improve how content share name is generated. (#18675)
+* : Fix an issue where adding VNET integration & Hybrid connections on a slot is not working (#18582)
+* `az appservice domain create`: Fix get correct domain agreements (#18622)
+* `az webapp deployment github-actions add/remove`: new commands (#18261)
+
+**AppConfiguration**
+
+* Add support for ```disable_local_auth``` (#18619)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:42]:
```suggestion
* Add support for `disable_local_auth` (#18619)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,136 @@
...
+
+* `az network routeserver create`: Add `--public-ip-address` argument. (#18663)
+
+**RDBMS**
+
+* Add autogrow parameter for MySQL and add database name to output json when created (#18441)
+
+**Resource**
+
+* Third party S2S Consent/Permission Enumeration (#18433)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:116]:
```suggestion
* Third-party S2S Consent/Permission Enumeration (#18433)
```

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,136 @@
...
+* Migrate ACS module to track 2 SDK (#18117)
+* Upgrade api-version to 2021-05-01 for ACS module (#18593)
+* Add UltraSSD support (#18649)
+* Support use custom kubelet identity (#18615)
+* `az aks get-credentials`: Add a check for KUBECONFIG environmental variable (#18704)
+
+**APIM**
+
+* Added version parameter for apim api import (#18604)
+* Fixed apim upgrade bug when specifying protocols (#18605)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:20]:
```suggestion
* Add version parameter for apim api import (#18604)
* Fix apim upgrade bug when specifying protocols (#18605)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,136 @@
...
+* `az apim create`: Fixed `--enable-managed-identity` true failure (#18554)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:21]:
```suggestion
* `az apim create`: Fix `--enable-managed-identity` true failure (#18554)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,136 @@
...
+
+**App Config**
+
+* Stop overwriting KeyVault reference content type during import (#18602)
+
+**App Service**
+
+* [BREAKING CHANGE] `az functionapp create`: Remove support for EOL Node 8 and 10 (#18676)
+* [BREAKING CHANGE] az webapp deployment source config: Remove vsts-cd-manager (#18203)
+* [BREAKING CHANGE] az functionapp deployment source config: Remove vsts-cd-manager (#18203)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:31]:
```suggestion
* [BREAKING CHANGE] `az webapp deployment source config`: Remove vsts-cd-manager (#18203)
* [BREAKING CHANGE] `az functionapp deployment source config`: Remove vsts-cd-manager (#18203)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,136 @@
...
+
+* Add support for ```disable_local_auth``` (#18619)
+
+**ARM**
+
+* `az provider register`: Make parameter `--accept-term` become not required (#18509)
+
+**ARO**
+
+* `az aro create`: add cidr values for pod/service (#18457)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:50]:
```suggestion
* `az aro create`: Add cidr values for pod/service (#18457)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,136 @@
...
+* Fail if resource doesn't exist on delete (#18546)
+
+**Azurestack**
+
+* [ACR][AKS] add azurestack hybrid profile for AKS (#18118)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:55]:
```suggestion
* [ACR][AKS] Add azurestack hybrid profile for AKS (#18118)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,136 @@
...
+
+**Backup**
+
+* Workload container registration fix, SDK upgraded to 0.12.0, Fixed and Re-ran tests (#18592)
+* Added Archive Support for Azure CLI (#18535)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:60]:
```suggestion
* Add Archive Support for Azure CLI (#18535)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,136 @@
...
+* `az aro create`: add cidr values for pod/service (#18457)
+* Fail if resource doesn't exist on delete (#18546)
+
+**Azurestack**
+
+* [ACR][AKS] add azurestack hybrid profile for AKS (#18118)
+
+**Backup**
+
+* Workload container registration fix, SDK upgraded to 0.12.0, Fixed and Re-ran tests (#18592)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:59]:
```suggestion
* `az backup container`: Fix container registration
Workload container registration fix, SDK upgraded to 0.12.0, Fixed and Re-ran tests (#18592)
```

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**